File: zapit-frontend/src/app/transaction/transaction-payment.ts
import {Transaction} from "./transaction";
import {User} from "../user/user";

export interface TransactionPayment {
    id: number;
    transaction: Transaction;
    amount: number;
    method: string;
    status: boolean;
    createdBy: User;
    createdOn: number;
    updatedOn: number;
}

File: zapit-backend/src/main/java/com/shaunwah/zapitbackend/model/TransactionPayment.java
package com.shaunwah.zapitbackend.model;

import lombok.*;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class TransactionPayment {
    private Long id;
    private Transaction transaction;
    private Double amount;
    private String method;
    private Boolean status;
    private User createdBy;
    private Long createdOn;
    private Long updatedOn;
}
